dart-ts-mode.el

```diff
@@ -289,6 +289,11 @@ PARENT is always optional_formal_parameters."
    :override t
    '((escape_sequence) @font-lock-escape-face)
 
+   :language 'dart
+   :feature 'function
```

Comment:
> Common feature names (for general programming languages) include
definition, type, assignment, builtin, constant, keyword,
string-interpolation, comment, doc, string, operator, property,
preprocessor, escape-sequence, key (in key-value pairs).

According to `treesit-font-lock-feature-list`, there is no `function`  feature. Please consider put "this" and "super" to another feature.

Reply:
Done.

Since they're both keywords, so I put them under `keyword` feature.